tasks/install-matlab/v1-beta/src/matlab.ts

```diff
@@ -0,0 +1,91 @@
+// Copyright 2023 The MathWorks, Inc.
+
+import * as toolLib from "azure-pipelines-tool-lib/tool";
+import * as fs from "fs";
+import * as https from "https";
+import * as script from "./script";
+
+export interface Release {
+    name: string;
+    version: string;
+    update: string;
+}
+
+export async function makeToolcacheDir(release: Release): Promise<[string, boolean]> {
+    let toolpath: string = toolLib.findLocalTool("MATLAB", release.version);
+    let alreadyExists = false;
+    if (toolpath) {
+        alreadyExists = true;
+    } else {
+        fs.writeFileSync(".keep", "");
+        toolpath = await toolLib.cacheFile(".keep", ".keep", "MATLAB", release.version);
+    }
+    return [toolpath, alreadyExists];
+}
+
+export async function getReleaseInfo(release: string): Promise<Release> {
+    // Get release name from input parameter
+    let name: string;
+    if ( release.toLowerCase().trim() === "latest") {
+        try {
+            name = await resolveLatest();
+        } catch {
+            return Promise.reject(Error(`Unable to retrieve the MATLAB release information. Contact MathWorks at continuous-integration@mathworks.com if the problem persists.`));

```

Comment:
```suggestion
            return Promise.reject(Error(`Unable to retrieve latest MATLAB release information. Contact MathWorks at continuous-integration@mathworks.com if the problem persists.`));

```

Reply:
Do we even need this? It looks like `resolveLatest` is throwing the same error message.

Reply:
You're right, not needed.